### DISCUSSION TITLE
CN 2.1 - blurry results

### DISCUSSION THREAD

**Discussion by blacklig** [lllyasviel/ControlNet#483] [Q&A]
Does anyone know why when using control nets for SD 2.1 I get always rather "blurry" and like "low quality jpeg" outputs? My prompts are good I think, also using some most famous negative embeddings, but sometimes not. No matter what I do, I cannot get crisp output of SD 2.1 (using models trained even on 832x832). If I disable CN for that particular settings, outputs are super clear, but with CN they tend to lose lot of that crispynes. Dont really know why. Has someone idea what might I be missing? Or is that just like  that that those models destroy quality? Interestingly on same settings with 1.5 I dont have those issues and I get crisp results (obviously different models for 1.5).

**Comment by geroldmeisinger** [comment]
make sure you use the correct controlnet model version with the correct stable diffusion model version. SD 1+CN 1 and  SD 2+CN 2 use different architectures

**Reply by blacklig** [reply]
Sure I had it like that.. Otherwise it would not even run I think, like pytorch or what throws errors etc.. Just couldnt get nice quality images from this.. dunno why, but I think I would focus on SDXL ..
